x-pack/plugins/security_solution_serverless/public/upselling/register_upsellings.tsx

```diff
@@ -40,7 +44,7 @@ interface UpsellingsMessageConfig {
   id: UpsellingMessageId;
 }
 
-type UpsellingPages = Array<UpsellingsConfig & { pageName: SecurityPageName }>;
+type UpsellingPages = Array<UpsellingsConfig & { pageName: SecurityPageName; path?: string }>;
```

Comment:
Is this needed?

Reply:
Good catch! Not needed at all. I think I added it for some reason and then forgot to clean it up

Reply:
b7cbf121e1fe70d8a8f35d83e951386a04f18866